### ISSUE TITLE
Questions about data

### ISSUE THREAD

**Issue by alvizek** [FlowWest/feather-mini-snorkel#2]
- [x] Confirm the in general methodology (ashley to prepare diagram)
- [x] Confirm that cover/substrate is being collected for each fish observation
- [x] Confirm the purpose of the microhabitat table as capturing habitat availability
- [x] What analysis is being done at the microhabitat level?
- [x] What is the rationale for using 1m2 in the microhabitat?
- [x] What are the canopy codings?
- [x] Confirm ERD (erin to prepare diagram) 
- [x] Is microhabitat & fish obs filtered the same as all fish? (there are 15 more records in the joined version) 
- [x] do the river units correspond to any of the variables?
- [x] do we have lat/long for the missing locations? I want to be able to associate observations to spatial location
- [x] confirm the scale of the substrate/cover observations?
- [x] Add Brad as contact on the EDI package

**Comment by alvizek** [comment]
Ashley and Ryon met on May 21. Ryon is going back to the datasheets to see if he can find better info about location and canopy cover. If not, we will likely use my location revised groups and associate a course coordinate.

**Comment by alvizek** [comment]
Ashley and Pascale met and decided it was pretty important to get location information. Ashley asked Ryon if at all possible to see if the paper field sheets had any additional location information. 

In the meantime, I QCd all the coordinates we had for the locations by searching in google, identifying any that didn't make sense, converting to lat/long. I then filled in missing coordinate information with coordinates we pulled from a kmz casey provided for the ongoing snorkel data or by averaging the lat/long by location_name and using the average to fill in gaps.

Badhia is working to put together a map of these lat/long colored by location name to see if these make sense.
